Week3/homework/js-exercises/creditCardValidator.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+{
+  // In this project you'll write a script
+  // that validates whether or not a credit card number is valid.
+  // Here are the rules for a valid number:
+  function checkCard(cardNumber) {
+
+    // Number must be 16 digits
+    if (cardNumber.length !== 16) {
+      console.log(`Card must be 16 digits!`);
+      return false;
+    }
+
+    // All of them must be numbers
+    for (let item = 0; item < cardNumber.length; ++item) {
+      let cardNumberItem = cardNumber[item];
+      cardNumberItem = Number.parseInt(cardNumberItem);
+
+      if (Number.isInteger(cardNumberItem) !== true) {
+        console.log(`Must be numbers all of them!`)
+        return false;
+      }
+    }
```

Comment:
How about just try to convert the `cardNumber` to a number using `parseInt` or `Number` in JavaScript instead of using a for loop? 🙂